VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -828,15 +851,26 @@ public Properties trackProfilePhotoSet(boolean fromCamera) {
 
     /**
      * This method sets category and labels to BI events
+     *
      * @param props
      * @param category
      * @param label
      * @return An updated {@link Properties} object with CATEGORY and LABEL
      */
-    private Properties addCategoryToBiEvents(Properties props, String category, String label){
+    private Properties addCategoryToBiEvents(Properties props, String category, String label) {
         props.put(Keys.CATEGORY, category);
         props.put(Keys.LABEL, label);
         return props;
     }
 
+    public static String getShareTypeFromComponentName(@NonNull ComponentName componentName) {
+        if (componentName.getPackageName().contains("facebook")) {
```

Comment:
Is there a more reliable way to do this? Like do the service names have known contracts or something?